transactions/testhelpers.go

```diff
@@ -0,0 +1,84 @@
+package transactions
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/stretchr/testify/mock"
+
```

Comment:
nit: is this standard practice to have all these gaps between imports? would be cleaner if not imo

Reply:
CI is back with error: `File is not `goimports`-ed with -local github.com/ethereum/go-ethereum,github.com/status-im/status-go,github.com/waku-org/go-waku (goimports)`

And my local `make lint` throws false positives:
```log
make lint
ln: /Users/stefan/proj/status/status-desktop/vendor/status-go//.git/hooks: Not a directory
lint
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused. 
WARN [runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused. 
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused. 
WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner. Replaced by revive. 
WARN [linters_context] structcheck is disabled because of generics. You can track the evolution of the generics support by following the https://github.com/golangci/golangci-lint/issues/2649. 
services/wallet/bridge/bridge.go:53:18: t.HopTx.From undefined (type *HopTxArgs has no field or method From) (typecheck)
		return t.HopTx.From
		               ^
services/wallet/bridge/bridge.go:55:22: t.CbridgeTx.From undefined (type *CBridgeTxArgs has no field or method From) (typecheck)
		return t.CbridgeTx.From
		                   ^
services/wallet/bridge/bridge.go:57:29: t.ERC721TransferTx.From undefined (type *ERC721TransferTxArgs has no field or method From) (typecheck)

```

I'm going back to separating the imports based on account